lib/http_server_fixture/handlers/options_two.ex

```diff
@@ -1,43 +1,20 @@
 defmodule HTTPServerFixture.OptionsTwo do
   alias HTTPServer.Request
-  alias HTTPServer.Response.Headers
-  import HTTPServer.Response.HeadersBuilder
   @behaviour HTTPServer.Handler
 
   @impl HTTPServer.Handler
-  def handle(%Request{method: "GET"} = req) do
+  def handle(%Request{method: "GET"} = _req) do
     body = ""
-
-    headers =
-      %Headers{}
-      |> content_length(body)
-      |> content_type()
-      |> host(req.headers)
-
-    {200, body, headers}
+    {200, body, :text}
   end
```

Comment:
These lean Handlers look great!  

User simply focuses on building the response body and the rest is left up to the server!

Reply:
I wonder if you could drop that status code too?